tests/network/primary_pod_network.go

```diff
@@ -82,6 +86,61 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			It("should report PodIP as its own on interface status", func() { AssertReportedIP(vmi) })
 		})
 	})
+
+	Describe("Connectivity", func() {
+		const servicePort = 1500
+		const ipv4HeaderSize = 20
+		const icmpHeaderSize = 8
+		const ipv4IcmpHeaderSize = ipv4HeaderSize + icmpHeaderSize
+
+		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance) {
+			inboundVMIAddress := inboundVMI.Status.Interfaces[0].IP
+
+			By("Checking connectivity via ICMP")
+			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress)).To(Succeed())
+
+			By("Checking connectivity via ICMP with MTU-sized frames")
+			mtu := getPodNetworkMTU(outboundVMI)
+			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress, "-c5", "-w5", "-s", strconv.Itoa(mtu-ipv4IcmpHeaderSize))).To(Succeed())
+
+			By("Checking connectivity via HTTP")
+			Expect(tests.PingHTTPFromVMConsole(outboundVMI, inboundVMIAddress, servicePort)).To(Succeed())
+		}
+
+		Context("2 VMIs connected to the pod network using the bridge binding", func() {
+			vmis := libvmi.NewVMIPool()
```

Comment:
I think it is not safe to call it here, I would prefer to call it in the BeforeEach, and just declare here the type here.